packages/frontend/src/components/VariablesList/index.tsx

```diff
@@ -1,79 +1,71 @@
-import type { ComponentType } from 'react'
-import List from '@mui/material/List'
-import ListItem, { ListItemProps } from '@mui/material/ListItem'
-import ListItemButton, {
-  ListItemButtonProps,
-} from '@mui/material/ListItemButton'
-import ListItemText from '@mui/material/ListItemText'
+import { Box, Text } from '@chakra-ui/react'
 import { type Variable } from 'helpers/variables'
 
-function makeListItemComponent(
+function makeVariableComponent(
   variable: Variable,
   onClick?: (variable: Variable) => void,
-): ComponentType<ListItemButtonProps> | ComponentType<ListItemProps> {
-  if (onClick) {
-    return (props: ListItemButtonProps) => (
-      <ListItemButton
-        {...props}
-        // onClick doesn't work sometimes due to latency between mousedown and immediate mouseup event after
-        onMouseDown={() => {
-          onClick(variable)
-        }}
-      />
-    )
-  }
-
-  return (props: ListItemProps) => <ListItem {...props} />
+): JSX.Element {
+  return (
+    <Box
+      key={`suggestion-${variable.name}`}
+      data-test="variable-suggestion-item"
+      padding={onClick ? '0.5rem 1rem' : '1rem'}
+      borderBottom={onClick ? undefined : '1px solid #EDEDED'}
+      _hover={
+        onClick
+          ? {
+              backgroundColor: 'secondary.50',
+              cursor: 'pointer',
+            }
+          : undefined
+      }
+      _active={
+        onClick
+          ? {
+              backgroundColor: 'secondary.100',
+              cursor: 'pointer',
+            }
+          : undefined
+      }
+      // onClick doesn't work sometimes due to latency between mousedown and immediate mouseup event after
+      onMouseDown={
+        onClick
+          ? () => {
+              onClick(variable)
+            }
+          : undefined
+      }
+    >
+      <Text textStyle="body-1" color="base.content.strong">
+        {variable.label ?? variable.name}
+      </Text>
+      <Text textStyle="body-2" color="base.content.medium">
+        {variable.displayedValue ?? variable.value?.toString() ?? ''}
+      </Text>
+    </Box>
+  )
 }
 
 interface VariablesListProps {
   variables: Variable[]
   onClick?: (variable: Variable) => void
-  listHeight?: number
 }
 
 export default function VariablesList(props: VariablesListProps) {
-  const { variables, onClick, listHeight } = props
+  const { variables, onClick } = props
 
   if (!variables || variables.length === 0) {
     return <></>
   }
 
   return (
-    <List
-      disablePadding
+    <Box
       data-test="variable-suggestion-group"
-      sx={{ maxHeight: listHeight, overflowY: 'auto' }}
+      h="256px"
```

Comment:
question: why this and not `fit-content`

Reply:
nvm I see why, cos this is shared.  I feel like we can expose height as a prop

Reply:
maybe something like

```
interface VariablesListProps {
  variables: Variable[]
  onClick?: (variable: Variable) => void
  height?: ComponentProps<typeof Box>['height']
}
```

Reply:
can i check if you think that a `maxH` would be sufficient?